Add security bulletin for H1 2025 Asgardeo (#121)
wso2/docs-security: en/docs/security-announcements/cloud-security-bulletins/asgardeo/2025/Asgardeo-2025-H1.md

> ---

Reviewer: Additionally, to map this document, 
1. Add an `index.md` file to maintain all the listing related to 2025 (refer to sample at https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cve-justifications/2025/index.md)
2. Need to update `mkdocs.yml` file to map all the pages (refer to CVE justification related references as a sample : https://github.com/wso2/docs-security/blob/main/en/mkdocs.yml#L347)

Reviewer 2: Yes please. 

[2025 Security Bulletins](https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cloud-security-bulletins/asgardeo/%7B%7B#base_path#%7D%7D/security-announcements/cloud-security-bulletins/asgardeo/2025) link is not working for now since we did not have this index.md 

Please add that file similar to https://github.com/wso2/docs-security/blob/main/en/docs/security-announcements/cve-justifications/2025/index.md

Reviewer 2: Could you please also add this to mkdocs.yml similar to what is done here to get the menu working https://github.com/wso2/docs-security/pull/117/files#diff-25b25bdf9af26b2bd51113914da350cc25a45c8d2c8a75cc3549ea8034e85de6

Why I'm trying to get all this done here is to make sure we can use the PR as an example for creating future bulletins. Thanks!

Author: Fixed with [16371c9](https://github.com/wso2/docs-security/pull/121/commits/16371c9cb14d5b1ddb634deccce5e10bcbfce1d7)

title: Asgardeo Security Bulletin – H1 2025
category: security-announcements
published: "2025-07-04"
version: "1.0.0"

> severity: "Multiple"

Reviewer: ```suggestion
```

---

# Asgardeo Security Bulletin – H1 2025

<p class="doc-info">Published: 2025-07-04</p>
<p class="doc-info">Version: 1.0.0</p>

> <p class="doc-info">Severity: Multiple</p>

Reviewer: ```suggestion
```

### BULLETIN ID  
ASG-SB-2025-H1

### SCOPE  

> This bulletin summarizes security vulnerabilities addressed during the month of January 2025 for Asgardeo.

Reviewer: ```suggestion
This bulletin summarizes security vulnerabilities addressed during the H1 of 2025 for Asgardeo.
```

| ASG-2025-004 | Clickjacking in Console and My Account | Medium | UI could be embedded via iframes, potentially tricking users into unintended actions. |
| ASG-2025-005 | Root org registration abuse | High | Attacker could register a root org using another user’s email and enumerate root orgs of unrelated users. |
| ASG-2025-006 | Username logged on auth failure | Low | Username values were exposed in logs during authentication failures. |
| ASG-2025-007 | Auth code not revoked on password update | Medium | Authorization codes continued to be valid after a user changed their password. |
| CVE-2025-24813 | Apache Tomcat RCE / Info Disclosure | Critical | Upstream vulnerability affecting embedded Tomcat. Patched to latest version. |
| ASG-2025-008 | NGINX “IngressNightmare” | Critical | Vulnerabilities in ingress configuration leading to potential unauthorized access or routing bypass. CVE-2025-1974, CVE-2025-24514, CVE-2025-1097, CVE-2025-1098, CVE-2025-24513 |
| ASG-2025-009 | Access Token retrieval via ‘code token’ flow | High | Improper validation allowed tokens to be issued under improper flow conditions. |

### CREDITS  

> Asgardeo thanks all internal and external researchers for responsibly disclosing the above issues.

Reviewer: All these are reported by external users?

Reviewer 2: +1 or at the `CREDITS` section, list down as follows.  It's unfair for them otherwise. 

- <reporter1> responsibly disclosed ASG-2025-001
- <reporter2> responsibly disclosed ASG-2025-002

Etc. This helps the table remain clean, when some rows do not have credit to be given.
